static/app/utils/profiling/hooks/useVirtualizedTree/useVirtualizedTree.tsx

```diff
@@ -87,6 +105,38 @@ export function VirtualizedTreeStateReducer<T>(
   }
 }
 
+function hideGhostRow({ref}: {ref: MutableRefObject<HTMLElement | null>}) {
+  if (ref.current) {
+    ref.current.style.opacity = '0';
+  }
+}
+
+function updateGhostRow({
+  ref,
+  tabIndexKey,
+  rowHeight,
+  scrollTop,
+  backgroundColor,
```

Comment:
I think this `updateGhostRow` should not take `backgroundColor` directly. There's quite a few references to the color that it use scattered all over the place because of this. Instead something like
```
mode: 'hover' | 'active'
```
would be more explicit about the intent and allow us to keep all the references to the color to use in one place.